### ISSUE TITLE
Couldn't get any accounts! Make sure your Ethereum client is configured correctly.

### ISSUE THREAD

**Issue by eastsee** [rajivjc/clinicaltrials-on-blockchain#2]
when get on the site 8081 after truffle serve, I get the above message and can not create any account like cro or pharma.

**Comment by rajivjc** [comment]
Do you have an Ethereum client like geth or testrpc running ? Have a look at this for more details -
https://github.com/ethereum/go-ethereum/wiki/Managing-your-accounts

**Comment by eastsee** [comment]
Yes. testrpc is running and ipfs daemon is running.  Truffle migrate works fine.
Since my first post,  I found your code is trying to connect to "localhost" at truffle.js and app.js
So, I have changed the files so that truffle looks my AwsServerIP:8545. 
However, I still have the same problem.

**Comment by rajivjc** [comment]
Do you have Metamask installed on your browser by any chance ?

**Comment by eastsee** [comment]
No. I don't.
Should I?

**Comment by deoidh** [comment]
@rajivjc cheers for the comment it is apparent that if you have MetaMask installed it will take the place of truffle's usual port + localhost.

**Comment by moranthomas** [comment]
I got this error too and the reason was a variation of the others already highlighted.

So I had tried to upgrade the MetaMask Chrome plugin and it was in the process of doing that - So MetaMask was enabled but I had not logged in.

Removing MetaMask from Chrome did the trick (and later I re-installed the upgraded version)

**Comment by moranthomas** [comment]
I got this issue too..  
So I had tried to upgrade the MetaMask Chrome plugin and it was in the process of doing that - So MetaMask was enabled and was trying to connect to the local blockchain on port 8545.

Removing MetaMask from Chrome did the trick (and later I re-installed the upgraded version)
